lang/image.bzl

```diff
@@ -202,8 +202,8 @@ def _app_layer_impl(ctx, runfiles=None, emptyfiles=None):
   symlinks += {
     # Create a symlink from our entrypoint to where it will actually be put
     # under runfiles.
-    _binary_name(ctx): "/".join([_reference_dir(ctx), ctx.label.package,
-                                 ctx.attr.binary.label.name]),
+    _binary_name(ctx): "/".join([_reference_dir(ctx),
+                                 ctx.attr.binary.files.to_list()[0].short_path]),
```

Comment:
This should probably just be:
```
_final_file_path(ctx.attr.binary.files.to_list()[0])
```

Can you confirm that things are broken by updating `rules_go` and fixed by applying this patch?

Reply:
Do you mean confirm by testing on my local machine? 
I am using rules_go 0.8.0 and without my fix I get this:
```
bazel run //cmd/feature:feature_image
INFO: Analysed target //cmd/feature:feature_image (0 packages loaded).
INFO: Found 1 target...
Target //cmd/feature:feature_image up-to-date:
  bazel-bin/cmd/feature/feature_image-layer.tar
INFO: Elapsed time: 0.352s, Critical Path: 0.00s
INFO: Build completed successfully, 1 total action

INFO: Running command line: bazel-bin/cmd/feature/feature_image
Loaded image ID: sha256:2bcba11e16b71463da26a30b08a7499592bbfd259fcc629eef2241d693049fd3
Tagging 2bcba11e16b71463da26a30b08a7499592bbfd259fcc629eef2241d693049fd3 as bazel/cmd/feature:feature_image
container_linux.go:265: starting container process caused "exec: \"/app/cmd/feature/feature\": stat /app/cmd/feature/feature: no such file or directory"
docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"/app/cmd/feature/feature\": stat /app/cmd/feature/feature: no such file or directory".```

With my fix it finds the executable when doing `bazel run`.

Reply:
Let me implement your suggestion and try again! One moment.

Reply:
Yeah, that's all I meant, thanks for confirming.

Reply:
Worked. I updated the code.